.pre-commit-config.yaml

```diff
@@ -3,4 +3,30 @@ repos:
     rev: 0.16.0
     hooks:
     - id: check-github-actions
-    - id: check-github-workflows
+    # TODO: reactivate when fixed
```

Comment:
What is this referring to? What needs to be fixed?

Reply:
https://gist.github.com/joshuarli/36587aeb18445bf18f36abcd4b82382e
(upgraded to 0.33.0 since in previous versions output is a lot less readable)

mostly complaining about `'uses' is a required property` which... it is not? lol